src/components/SetupOpportunity/SetupOpportunity.tsx

```diff
@@ -76,6 +71,7 @@ export const SetupOpportunity: FC<Props> = ({ opportunity }) => {
             </Details>
             <WorkflowComponentList
                 websiteListings={hasWebsiteListings ? websiteListings : null}
+                applications={hasApplications ? applications : null}
```

Comment:
Just pass websiteListings, which will default to undefined, and change the typings for WorkflowCOmponentList.

Reply:
Fixed